docker/lr-10x/Dockerfile

```diff
@@ -58,8 +78,21 @@ ADD reverse_adapter_sequence.fasta.bwt /lrma/reverse_adapter_sequence.fasta.bwt
 ADD reverse_adapter_sequence.fasta.pac /lrma/reverse_adapter_sequence.fasta.pac
 ADD reverse_adapter_sequence.fasta.sa /lrma/reverse_adapter_sequence.fasta.sa
 
+RUN pip3 install pysam biopython
```

Comment:
we really need to tidy up these environments. Mixing pip and conda isn't good.

Reply:
I added an issue (#349) to track this.